test/integration/users_test.rb

```diff
@@ -1,44 +0,0 @@
-require 'test_helper'
```

Comment:
@sunilkr Why are we deleting this file?

Reply:
Renamed users_test -> user_password_test to be consistent with controller. Other user paths are already covered in devise_registration_test, devise_login_test. Git didn't treat as file rename.